Bundle/WidgetBundle/Model/WidgetManager.php

```diff
@@ -124,6 +130,9 @@ public function newWidget($mode, $type, $slot, $view, $position, $parentWidgetMa
         $widget = $this->widgetHelper->newWidgetInstance($type, $view, $slot, $mode);
         $widgets = [$widget];
 
+        //parent container ids
+        $containerIds = explode(',', $this->getRequest()->get('containerIds', null));
```

Comment:
This line will probably be useless if you let php generate the route for the action instead of doing it with the toString javascript fn